frontend/server/src/Controllers/User.php

```diff
@@ -893,6 +893,16 @@ public static function apiGenerateOmiUsers(\OmegaUp\Request $r): array {
             $keys =  [
                 'Pr8oUAIE' => 20
             ];
+            elseif ($r['contest_type'] == 'OMICHH') {
```

Comment:
esto tiene un error de sintaxis:

```suggestion
        } elseif ($r['contest_type'] == 'OMICHH') {
```